src/main/java/firsttask/BubbleSort.java

```diff
@@ -1,7 +1,7 @@
 package firsttask;
 
 public class BubbleSort {
-    public static void usingBubbleSort(int[] arr) {
+    public static int[] usingBubbleSort(int[] arr) {
         for (int i = 0; i < arr.length - 1; i++) {
```

Comment:
does the task with bubble sort still work after the changes made here?

Reply:
Sure.
The output.txt file contains the following: "The result is: [-89, -78, -9, 0, 1, 7, 9, 17, 23, 55, 87, 333]"